package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "I hate webpack",
+  "private": true,
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
+  },
```

Comment:
- [ ] Kindly ensure that `webpack-dev-server` is installed under the `devDependencies` in `package.json` for this line to execute.
- [ ] For this line `webpack-dev-server --mode development --open` Kindly change the `webpack-dev-server` to `webpack serve`. It should the read `webpack serve --mode development --open`. This is one of the recent changes that was made to webpack.